etc/input.conf

```diff
@@ -123,10 +123,12 @@
 #S screenshot video                     # ...without subtitles
 #Ctrl+s screenshot window               # ...with subtitles and OSD, and scaled
 #Alt+s screenshot each-frame            # automatically screenshot every frame
-#w add panscan -0.1                     # zoom out with -panscan 0 -fs
-#e add panscan +0.1                     #      in
+#w add video-zoom -0.1                     # zoom out with -video-zoom 0 -fs
+#e add video-zoom +0.1                     #      in
```

Comment:
Any reason for this? I use panscan pretty much exclusively to watch youtube videos with messed up black bars, so making it constrained by the screen edges is something I appreciate